lib/defra_ruby/validators/companies_house_service.rb

```diff
@@ -34,16 +36,30 @@ def json_response
 
       private
 
+      def validate_permitted_types
+        return if @permitted_types.nil?
+
+        return if @permitted_types.is_a?(String) || @permitted_types.is_a?(Array)
+
+        raise ArgumentError, "Invalid permitted_types value - it must be nil, a string or an array of strings"
```

Comment:
it should be
`I18n.t("defra_ruby.validators.CompaniesHouseNumberValidator.argument_error")`
instead of 
`"Invalid permitted_types value - it must be nil, a string or an array of strings"`

Reply:
Thanks - fixed